src/components/scroll/ScrollToTop.js

```diff
@@ -0,0 +1,39 @@
+import { isVisible } from "@testing-library/user-event/dist/utils";
```

Comment:
add this library too in the package.json i think because of this depolyment is failing